docs/development/setup_development.rst

```diff
@@ -29,12 +29,23 @@ Ubuntu or Debian GNU/Linux
    sudo apt-get update
    sudo apt-get install -y make git
 
+
+Fedora and Other Linux Distributions
```

Comment:
For consistency's sake, if we're adding a Fedora section for the dev environment setup we should consider adding it in the staging environment as well. That does get more complicated as there's all the virtualbox and/or libvirt setup to consider, though.

Reply:
In the spirit of incremental improvements, adding a note about Fedora to the Docker instructions is IMO a good first step.